targettests/execution/cudaq_observe.cpp

```diff
@@ -8,6 +8,8 @@
 
 // REQUIRES: c++20
 // clang-format off
+// RUN: nvq++ --target anyon                    --emulate %s -o %t && %t | FileCheck %s
```

Comment:
Please remove 'anyon' owing to issue# https://github.com/NVIDIA/cuda-quantum/issues/2249

Reply:
Fixed in 2543b967b60011011323595ed6b27259563b537e